python/observability-online-eval/agno/dockerfile

```
@@ -0,0 +1,26 @@
+# Use an official Python image
+FROM python:3.10-slim
+
+# Install system dependencies (if needed)
+RUN apt-get update && apt-get install -y build-essential curl
+
+# Install uv
+RUN curl -Ls https://astral.sh/uv/install.sh | sh
```

Hadolint (2.12.0)

[warning] Set the SHELL option -o pipefail before RUN with a pipe in it. If you are using /bin/sh in an alpine image or if your shell is symlinked to busybox then consider explicitly setting your SHELL to /bin/ash, or disable this check

(DL4006)